test/ExtInst.OpenCL.std.cpp

```diff
@@ -152,8 +152,8 @@ INSTANTIATE_TEST_CASE_P(
         CASE2(Modf, modf),
         CASE1(Nan, nan),
         CASE2(Nextafter, nextafter),
-        CASE3(Pow, pow),
-        CASE1(Pown, pown),
+        CASE2(Pow, pow),
+        CASE2(Pown, pown),
```

Comment:
Seems to violate the [spec](https://www.khronos.org/registry/spir-v/specs/1.0/OpenCL.ExtendedInstructionSet.100.html).